tests/test_serialization.py

```diff
@@ -37,6 +38,20 @@ def save_load_from_filename(conf: Container, resolve: bool, expected: Any) -> No
         os.unlink(fp.name)
 
 
+def save_load_from_pathlib_path(conf: Container, resolve: bool, expected: Any) -> None:
```

Comment:
instead of duplicating the other function please parametrize it with pytest.mark.parametrize()

Reply:
Thanks @omry.

I was able to implement the other suggestions, and while I would like to keep things DRY, I'm not quite sure how to parametrise using the original save_load_from_filename. Could you please point me in the right direction?

Reply:
Made an implementation. Hopefully this is reasonably elegant :)

Reply:
It's good, and trickier than usual because you can't construct the object outside due to the temp file.
good job :).